gradle/libs.versions.toml

```diff
@@ -9,6 +9,7 @@ kotlinx-coroutines = "1.5.2"
 kotlin-wrappers = "0.0.1-pre.247-kotlin-1.5.31"
 spring-boot = "2.6.0"
 spring-security = "5.6.0"
+spring-cloud = "3.1.0-M3"
```

Comment:
This weird version is because spring-cloud with support for spring-boot 2.6.x is not released yet 🤷

Reply:
It's released